service-level-indicators/spark-retrieval-success-rate.md

```diff
@@ -0,0 +1,612 @@
+    - [**How many tasks should each checker do per round? What value is given to $k$?**](#how-many-tasks-should-each-checker-do-per-round-what-value-is-given-to-k)
+  - [Retrieval Checks](#retrieval-checks)
+  - [Reporting Measurements to Spark-API](#reporting-measurements-to-spark-api)
+  - [Evaluating Measurements with Spark-Evaluate](#evaluating-measurements-with-spark-evaluate)
+  - [On-chain Evaluation and Rewards](#on-chain-evaluation-and-rewards)
+- [Spark RSR Calculations](#spark-rsr-calculations)
+  - [High level Spark RSR Figure](#high-level-spark-rsr-figure)
+  - [Spark RSR by SP](#spark-rsr-by-sp)
+- [Appendix](#appendix)
+  - [Terminology](#terminology)
+    - [Clients](#clients)
+    - [Checker / Checker Node](#checker--checker-node)
+    - [Committee](#committee)
+    - [Eligible Deal](#eligible-deal)
+    - [**Retrieval Task**](#retrieval-task)
+    - [Round Retrieval Task List](#round-retrieval-task-list)
+    - [Retrieval Result](#retrieval-result)
+    - [Retrieval Task Measurement](#retrieval-task-measurement)
+    - [Committee Retrieval Task Measurements](#committee-retrieval-task-measurements)
+    - [Accepted Retrieval Task Measurement](#accepted-retrieval-task-measurement)
+    - [Committee Accepted Retrieval Task Measurements](#committee-accepted-retrieval-task-measurements)
+    - [Provider Retrieval Result Stats](#provider-retrieval-result-stats)
+  - [Callouts/Concerns with this SLI](#calloutsconcerns-with-this-sli)
+  - [Retrieval Result Mapping to RSR](#retrieval-result-mapping-to-rsr)
+  - [Per Request (non-committee) Score vs. Committee Scoring](#per-request-non-committee-score-vs-committee-scoring)
+  - [FAQ](#faq)
+    - [Why is graphsync used?](#why-is-graphsync-used)
+    - [What improvements are planned for Spark retrieval checking?](#what-improvements-are-planned-for-spark-retrieval-checking)
+    - [Why do checkers report back the storage clients that made the deal?](#why-do-checkers-report-back-the-storage-clients-that-made-the-deal)
+    - [Why do IPNI outages impact SP RSR?](#why-do-ipni-outages-impact-sp-rsr)
+
+
+# Meta
+
+## Document Purpose
+
+This document is intended to become the canonical resource that is referenced in [the Storage Providers Market Dashboard](https://github.com/filecoin-project/filecoin-storage-providers-market) wherever the “Spark Retrievability” graphs are shown.  A reader of those graphs should be able to read this document and understand the “Spark Retrievability SLO”.  The goal of this document is to explain fully and clearly “the rules of the game”.  With the “game rules”, we seek to empower market participants - onramps, aggregators and Storage Providers (SPs) - to “decide how they want to play the game”.
+
+## Versions
+
+TODO: add a table that shows Spark RSR version 1 is supported by which backend component versions.
+When we have v1.1, this would be a new row in a table, with a new set of backend cmponents.  
```

Comment:
I'm not sure how to answer this - and where Spark RSR version 1 even starts. Shall we say that the current service git commits are the first ones?

Reply:
I think it's fine to say that the current service commits are the first ones.